Cargo.toml

```diff
@@ -59,7 +59,7 @@ futures-timer = { version = "3.0" }
 hash-db = { version = "0.16", default-features = false }
 hex = { version = "0.4" }
 hex-literal = { version = "0.3" }
-ink = { version = "5.0.0-rc" }
+ink = { version = "5.0.0-rc", default-features = false }
```

Comment:
There's already 5.0.0-rc3 ([crates](https://crates.io/crates/ink/versions)) but I'd be wary of using RC at all.

Reply:
not related here, but +1

I will bump this once release of ink5 is concluded